CHANGELOG.yml

```
@@ -15,12 +15,7 @@ upcoming:
     - Add artwork Material filter - dzmitry tratsiak
     - Prevent android dark mode forced on users - kizito
     - Fix unusable slider on android - dzmitry
-    - Add search UI for material filter - dzmitry tratsiak
-    - Implements custom size filter with localization (behind feature flag) - damon
-    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
-    - Add order history to user profile (behind feature flag) - alexj105
-    - Enable lotsByFollowedArtists - ole
-    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
+    - Fix vanishing artist series - mounir
```

```suggestion
    - Fix vanishing artist series - mounir
    - Add search UI for material filter - dzmitry tratsiak
    - Implements custom size filter with localization (behind feature flag) - damon
    - Adds artist nationality & ethnicity filter (behind feature flag) - damon
    - Add order history to user profile (behind feature flag) - alexj105
    - Enable lotsByFollowedArtists - ole
    - Add ability to unselected "ways to buy" filter - dzmitry tratsiak
```
OMG I HATE CHANGELOG CONFLICTS

---

soon we won't have to deal with them 🙌